lib/model/model.go

```diff
@@ -162,6 +164,8 @@ func NewModel(cfg *config.Wrapper, id protocol.DeviceID, clientName, clientVersi
 		remotePausedFolders: make(map[protocol.DeviceID][]string),
 		fmut:                sync.NewRWMutex(),
 		pmut:                sync.NewRWMutex(),
+
+		scannerLimiter: scanner.NewScannerLimiter(cfg.Options().SingleGlobalScanner),
```

Comment:
Lets not do the boolean parameter. We don't have other types that takes a boolean for whether they do their thing or not. If needed, we `if` here and set the attribute to some no-op variant of the interface here.